Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -2557,77 +2578,134 @@ private void HandleTickSize(object sender, IB.TickSizeEventArgs e)
         {
             Symbol symbol;
 
-            if (!_subscribedTickets.TryGetValue(e.TickerId, out symbol)) return;
+            if (!_subscribedTickets.TryGetValue(e.TickerId, out symbol))
```

Comment:
An idea: Maybe `_subscribedTickets` could hold the `Symbol` and all `Tick` types in a new class, would help us avoid the extra `ConcurrentDictionaries`, their access and the `if(!) add` logics, because since we are not removing them, after the first tick the symbol will already be in the dictionaries ?

Reply:
Good idea, will do 👍